From e4f9128a5580ed53b62640a3a754d8fc58d04337 Mon Sep 17 00:00:00 2001
From: Henrik Pihl <ahvenas@gmail.com>
Date: Sat, 11 Jul 2020 11:43:50 +0300
Subject: [PATCH] Update org.shotcut.Shotcut.desktop

XDG standards tell that comment fields in the desktop files shouldn't be redundant. My Russian and French aren't good enough to replace those fields with something else.
---
 packaging/linux/org.shotcut.Shotcut.desktop | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/packaging/linux/org.shotcut.Shotcut.desktop b/packaging/linux/org.shotcut.Shotcut.desktop
index a9cf44da2..c5a4eff9e 100644
--- a/packaging/linux/org.shotcut.Shotcut.desktop
+++ b/packaging/linux/org.shotcut.Shotcut.desktop
@@ -5,10 +5,8 @@ GenericName=Video Editor
 GenericName[de]=Video-Bearbeitungsprogramm
 GenericName[fr]=Logiciel de montage vidéo
 GenericName[ru]=Видеоредактор
-Comment=Video Editor
+Comment=Shotcut is a free, open source, cross-platform video editor.
 Comment[de]=Programm zum Bearbeiten und Abspielen von Videodateien.
-Comment[fr]=Logiciel de montage vidéo
-Comment[ru]=Видеоредактор
 Exec=shotcut %F
 Icon=org.shotcut.Shotcut
 Terminal=false
